Formula/minio.rb

```diff
@@ -1,17 +1,11 @@
 class Minio < Formula
   desc "object storage server compatible with Amazon S3"
   homepage "https://github.com/minio/minio"
-  url "https://github.com/minio/minio/archive/RELEASE.2016-06-03T19-32-05Z.tar.gz"
-  version "20160603193205"
-  sha256 "8ecf52cc344c99f84bef632b8dfecf8fdc6a8493a541b2fe42f5b22d8bbfa802"
-  head "https://github.com/minio/minio.git"
+  url "https://github.com/minio/minio/archive/RELEASE.2016-07-13T21-46-05Z.tar.gz"
+  version "2016-07-13T21:46:05Z"
```

Comment:
This is interpreted as an earlier version than `20160603193205`, so `brew upgrade` will skip it.  Can you stick with the original version schema?